rubicon_ml/client/artifact.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, domain: ArtifactDomain, parent: Project):
     def _get_data(self):
         """Loads the data associated with this artifact."""
         project_name, experiment_id = self.parent._get_identifiers()
-        return_err = None
+        return_err = RuntimeError("Data was not loaded properly.")
```

Comment:
`return_err` is to re-raise errors from the repository layer - if there is an error this will always be overwritten in the `except` block below